lib/Rx/Disposable/SerialDisposable.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: matt
+ * Date: 9/15/15
+ * Time: 5:22 PM
+ */
+
+namespace Rx\Disposable;
+
+use Rx\DisposableInterface;
+
+class SerialDisposable implements DisposableInterface
+{
+    private $isDisposed = false;
```

Comment:
Yeah, I mean add them for properties/methods in order to "kind-of-constraint" types.